pkg/pkgmgr/rpm.go

```diff
@@ -113,15 +113,32 @@ func isLessThanRPMVersion(v1, v2 string) bool {
 // Map the target image OSType & OSVersion to an appropriate tooling image.
 func getRPMImageName(manifest *unversioned.UpdateManifest, osType string, osVersion string, useCachePrefix bool) string {
 	var image, version string
+	imagePrefix := "mcr.microsoft.com"
+	useCache := useCachePrefix
 
-	if osType == utils.OSTypeAzureLinux {
+	switch {
+	case osType == utils.OSTypeAzureLinux:
 		image = "azurelinux/base/core"
 		if strings.Contains(osVersion, "3.0") {
 			version = "3.0"
 		} else {
 			version = osVersion
 		}
-	} else {
+	case utils.IsSUSEImage(osType):
+		imagePrefix = "registry.opensuse.org"
+		version = osVersion
+		useCache = false
```

Comment:
I assume this is because we currently don't mirror OpenSUSE images to `ghcr.io`? I can open an issue to solve that in another PR!

Reply:
Makes sense in order to keep consistency with the other tooling images. Can you point me to the place that I can add the mirror config, please?

Reply:
Fixed.